example/main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"net/http"
 	"sort"
 
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
what is this for?

Reply:
there is a lot going on in this last commit. it'd be great to split it up into a few different changes - maybe by valuer/scanner/livesql.

Reply:
This is in order to "install" the mysql driver.